lib/uniris/oracles/oracles/coingecko.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Uniris.Oracles.Coingecko do
```

Comment:
I think a name as `Uniris.Oracles.UCOPrice.Coingecko` will make more sense as the CoinGecko is not an Oracle by itself